packages/transaction-controller/src/TransactionController.ts

```diff
@@ -963,6 +968,63 @@ export class TransactionController extends BaseController<
     }
   }
 
+  updateTransactionGasFees(
+    transactionID: string,
```

Comment:
```suggestion
    transactionId: string,
```

Reply:
For the sake of consistency, I've applied this to whole file here: https://github.com/MetaMask/core/pull/1674/commits/67f26a4a45e174e6bcec3b6e2cf983f29e38b383